docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -53,7 +59,7 @@ To provide the best possible environment for Strapi there are a few requirements
 
 #### 1. Configure
 
-We always recommend you use environment variables to configure your application based on the environment. Here is an example:
+We recommend using environment variables to configure your application based on the environment, for example:
```

Comment:
Elsewhere the file paths have been moved inside the code blocks, but I know this was an area of recent discussion about which is preferable.

Reply:
We indeed have the file path inside the code block as a comment.
We might eventually revert this change, introduced roughly a year ago, as some people don't like it. But for now and for consistency, I'd keep the path inside the code block as Shaun suggested.